components/Players/PlayerManager.vue

```diff
@@ -348,11 +370,7 @@ export default Vue.extend({
       this.setNextTrack();
     },
     togglePause(): void {
-      if (this.isPaused) {
-        this.unpause();
-      } else {
-        this.pause();
-      }
+      this.playPause();
     }
```

Comment:
Maybe map the `playPause` action to the component and directly call it from the different buttons?

Reply:
I thought about that, but couldn't be bothered to do it lol. I've updated it 👍